airflow-core/src/airflow/serialization/serialized_objects.py

```diff
@@ -2663,7 +2663,7 @@ def conversion_v1_to_v2(ser_obj: dict):
             "_operator_extra_links",
         ]
 
-        ser_obj["__version"] = 2
+        ser_obj["__version"] = 3
```

Comment:
Nit: the fn still says v1 to v2. Change this or the fn name. My pref is this
```suggestion
        ser_obj["__version"] = 2
```

Reply:
Whoops, yeah

Reply:
Fixed in https://github.com/apache/airflow/pull/55836/commits/05f2ec7aa451b7a12ee2a183d9f1799be596bf5a